src/entries/popup/Routes.tsx

```diff
@@ -993,6 +994,10 @@ const RootLayout = () => {
 
   React.useEffect(() => {
     analytics.screen(screen[pathname], { path: pathname });
+    popupClient.analytics.addRouterBreadcrumb({
+      path: pathname,
+      params: { name: screen[pathname], state },
```

Comment:
I think `state` could include some password/wallet data, so stripped it; can revisit if we think we need it. Added some TODOs on where it seemed inappropriate and leak prone